.github/workflows/build_napari.yml

```diff
@@ -85,12 +85,9 @@ jobs:
           # problems with screenshots (https://github.com/napari/docs/issues/285)
           linux-setup: "echo 'skip setup'"
           linux-teardown: "echo 'skip teardown'"
-    
-      - name: remove .doctrees folder (400Mb)
-        run: rm -rf napari-docs/docs/_build/.doctrees
 
       - name: Upload artifact
         uses: actions/upload-artifact@v4
         with:
           name: napari-docs
-          path: napari-docs/docs/_build
\ No newline at end of file
+          path: napari-docs/docs/_build/html/
```

Comment:
Unfortunately we don't do that - we could though!

After build, this is the folder structure:

```
ls docs/_build 
api         _downloads         gallery.html   howtos      _modules     plugins           roadmaps        sg_execution_times.html  _static    usage.html
community   further-resources  genindex.html  _images     naps         py-modindex.html  search.html     _sources                 _tags
developers  gallery            guides         index.html  objects.inv  release           searchindex.js  _sphinx_design_static    tutorials
```

To fix that, we'd have to fix the docs Makefile to read

```
sphinx-build -b html docs/ docs/_build/html
```

instead of 

```
sphinx-build -b html docs/ docs/_build 
```

Reply:
Hmm, that was one of the changes in the Unify workflows PR:
https://github.com/napari/docs/pull/348/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52
Using -M
```
The default output directory locations when using make-mode differ from the defaults when using [-b](https://www.sphinx-doc.org/en/master/man/sphinx-build.html#cmdoption-sphinx-build-b).
- doctrees are saved to <outputdir>/doctrees
- output files are saved to <outputdir>/<builder name>
```

I will double check all this later today.

Reply:
Ah! I may just have an outdated version of the repo then, I will check